lib/src/main/java/uk/gov/di/ipv/core/library/validation/TokenRequestValidator.java

```diff
@@ -94,14 +106,11 @@ private ClientAuthenticationVerifier<Object> getClientAuthVerifier(
     }
 
     private void logWarningJtiHasAlreadyBeenUsed(ClientAuthJwtIdItem clientAuthJwtIdItem) {
-        LoggingUtils.appendKey(
-                LogHelper.LogField.JTI_LOG_FIELD.getFieldName(), clientAuthJwtIdItem.getJwtId());
-        LoggingUtils.appendKey(
-                LogHelper.LogField.JTI_USED_AT_LOG_FIELD.getFieldName(),
-                clientAuthJwtIdItem.getUsedAtDateTime());
-        LOGGER.warn("The client auth JWT id (jti) has already been used");
-        LoggingUtils.removeKeys(
-                LogHelper.LogField.JTI_LOG_FIELD.getFieldName(),
-                LogHelper.LogField.JTI_USED_AT_LOG_FIELD.getFieldName());
+        var message =
+                new MapMessage()
+                        .with("jwtValidation", "The client auth JWT id (jti) has already been used")
```

Comment:
Either the name of these map fields can use the enum values from the LogHelper.LogField enum, or those values can be removed as they aren't being used now.

Reply:
Yeah, I'll clear them up.

Reply:
I removed the LogHelper enums